packages/python/README.md

```diff
@@ -44,19 +45,19 @@ Ex)
 MetricsApiConfig(
     README_API_KEY,
     lambda req: {
-        'id': 'unique id of user making call',
+        'api_key': 'unique api_key of the user',
         'label': 'label for us to show for this user (ie email, project name, user name, etc)',
         'email': 'email address for user'
     },
     buffer_length=1,
-    blacklist=['credit_card'] # Prevents credit_card in the request from being sent to readme
+    denylist=['credit_card'] # Prevents credit_card in the request from being sent to readme
 )
 ```
 
-| Option | Use |
-| :--- | :--- |
-| development_mode | **default: false** If true, the log will be separate from normal production logs. This is great for separating staging or test data from data coming from customers |
-| blacklist | **optional** An array of keys from your API requests and responses headers and bodies that you wish to blacklist from sending to ReadMe.<br /><br />If you configure a blacklist, it will override any whitelist configuration. |
-| whitelist | **optional** An array of keys from your API requests and responses headers and bodies that you only wish to send to ReadMe. |
-| buffer_length | **default: 10** Sets the number of API calls that should be recieved before the requests are sent to ReadMe |
-| allowed_http_hosts | A list of allowed http hosts for sending data to the ReadMe API.|
+| Option             | Use                                                                                                                                                                                                                           |
+| :----------------- | :---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| development_mode   | **default: false** If true, the log will be separate from normal production logs. This is great for separating staging or test data from data coming from customers                                                           |
+| denylist           | **optional** An array of keys from your API requests and responses headers and bodies that you wish to denylist from sending to ReadMe.<br /><br />If you configure a denylist, it will override any allowlist configuration. |
```

Comment:
```suggestion
| denylist           | **optional** An array of keys from your API requests, responses headers, and request bodies that you wish to not be sent to ReadMe.<br /><br />If you configure a denylist, it will override any allowlist configuration. |
```